cmd/gateway-main.go

```diff
@@ -125,6 +125,22 @@ EXAMPLES:
 
 `
 
+const siaGatewayTemplate = `NAME:
+  {{.HelpName}} - {{.Usage}}
+
+USAGE:
+  {{.HelpName}} {{if .VisibleFlags}}[FLAGS]{{end}}
+{{if .VisibleFlags}}
+FLAGS:
+  {{range .VisibleFlags}}{{.}}
+  {{end}}{{end}}
+
+EXAMPLES:
+  1. Start minio gateway server for GCS backend.
```

Comment:
Fix the GCS should say Sia backend here.. and perhaps also should list out an example setup here.

Reply:
Corrected. Also added all ENV variables and descriptions and defaults to the printout message.